libs/vscode-ui/components/src/lib/multiple-select/multiple-select.component.html

```diff
@@ -1,20 +1,21 @@
 <div class="description">
   {{ field.description }}
 </div>
-
-<select
-  [formControl]="selectControl"
-  multiple
-  role="combobox"
-  [id]="field.name"
-  [required]="field.required"
-  [disabled]="disabled"
```

Comment:
Removed the disabled attribute and @Input() to clear the ReactiveForms warning about setting it on the FormControl instead. I can't see any reason why these need to support disabled right now.